### PR TITLE
Added peru package.

### PR DESCRIPTION
**Pull request by jraygauthier**
A project dependencies version management system
built on top of multiple version management systems.

### CODE REVIEW

File: pkgs/applications/version-management/peru/default.nix [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  version = "0.1.3"; 
+in
+
+python3Packages.buildPythonPackage rec {
+  
+  # Do not prefix name with python specific version identifier.
+  namePrefix = "";
+
+  name = "peru-${version}";
+  src = fetchurl {
```
Comment (jagajaga) [inline pkgs/applications/version-management/peru/default.nix]: Better use `fetchFromGithub`.

Comment (jraygauthier) [inline pkgs/applications/version-management/peru/default.nix]: Nice to know it exists. Will have a look at it and update the PR.

**Comment by Fuuzetsu** [comment]
On top of the `fetchFromGitHub` comment that remains to be addressed, `peru` itself is on `0.2.x` now. Please open a fresh PR with an updated version if you're still interested.

**Comment by jraygauthier** [comment]
Ok. Will do.

**Comment by jagajaga** [comment]
Rebase and force push here again.

**Comment by Fuuzetsu** [comment]
@jagajaga normally I agree with the no new PR stuff but when stuff is months old, many people just don't have the interest in the content anymore and opening a new PR doesn't hurt.

**Comment by jraygauthier** [comment]
I'm simply not working with an up-to-date enough version of nixpkgs to perform what is requested here.
This `fetchFromGithub` is a good proposition, but it simply does not exist in my version of nixpkgs.

**Comment by jraygauthier** [comment]
@Fuuzetsu: Done as requested (github specific fetch + latest version).

**Pull request merged**